scripts/configs/mainnet.config.ts

```diff
@@ -60,3 +60,16 @@ export const mantleChainConfiguration: DeployNetworkConfig = {
     finalityDelaySeconds: 604800,
   },
 }
+
+export const curtisTestnetChainConfiguration: DeployNetworkConfig = {
+  chainId: networks.curtisTestnet.chainId, // chainId
+  chainConfiguration: {
+    provingMechanism: networks.curtisTestnet.proving.mechanism, // provingMechanism
+    settlementChainId: networks.curtisTestnet.proving.settlementChain.id, // settlementChainId
+    settlementContract: networks.curtisTestnet.proving.settlementChain.contract, // settlementContract e.g DisputGameFactory or L2OutputOracle.
+    blockhashOracle: networks.curtisTestnet.proving.l1BlockAddress, // blockhashOracle
+    outputRootVersionNumber:
+      networks.curtisTestnet.proving.outputRootVersionNumber, // outputRootVersionNumber
+    finalityDelaySeconds: 0,
+  },
+}
```

Comment:
This `networks` variable is for Mainnet and curtisTestnet is defined in the [testnet config file](https://github.com/ConstellationCrypto/eco-routes/pull/1/files#diff-9ef9ad6e05c97e446f53d15ae5ebb84ab48a2b082376d89533436f012d327f0dR266-R292)